dask_match/tests/test_collection.py

```diff
@@ -323,6 +325,9 @@ def test_tree_repr(df):
     assert "None" not in s
     assert "skipna=False" in s
     assert str(df.seed) in s.lower()
+    if fuse:
+        assert "Fused" in s
+        assert s.count("|") == 9
```

Comment:
Nit: This exact count seems a little brittle to me.  I could imagine changes to the code that changed this count but wouldn't necessarily mean an error (maybe we get more verbose in our number of terms).  

I'm going to go ahead and merge, mostly I mention this to align on style over time.